README.md

```diff
@@ -207,6 +207,40 @@ This operation requires a payload of the following form:
 
 Get the list of all entries that have been bookmarked by the currently authenticated user. Calling this as anonymous user yields an object with property `count` equals to `0`.  As a base URL call this returns an HTML page with formatted result, as url with `?format=json` suffix this results a JSON object for use as data input to applications, webpages, etc.
 
+### `GET /api/pulse/profiles/<id=number>/` with optional `?format=json`
+
+This retrieves a single user profile with the indicated `id` as stored in the database. Any profile can be retrieved using this route even without being authenticated. As a base URL call this returns an HTML page with formatted results, as url with `?format=json` suffix this results a JSON object for use as data input to applications, webpages, etc.
+
+### `GET /api/pulse/myprofile/` with optional `?format=json`
+
+This retrieves the user profile for the currently authenticated user as stored in the database. An unauthenticated user will receive an HTTP 403 Forbidden response if they try to access this route. As a base URL call this returns an HTML page with formatted results, as url with `?format=json` suffix this results a JSON object for use as data input to applications, webpages, etc.
+
+### `PUT /api/pulse/myprofile/`
+
+Allows an authenticated user to update their profile data. The payload that needs to be passed into this PUT request is:
+
+```
+{
+  user_bio: optional string (max length 140 characters)
+  custom_name: optional string containing the user's alternative name (max length 500 characters)
+  is_group: optional boolean indicating whether this profile belongs to a group of people or not (defaults to false)
```

Comment:
Sorry @gideonthomas can you explain what "group" we are referring to here? It might be just me but I don't understand what this boolean is for. Is `is_group` an indicator to tell if an user belongs to "the Network"???

Reply:
A profile can represent an organization or group of people. These profiles would not represent a single user, but could be 'owned' by a user. We won't do anything significant with this right now (and it doesn't impact your frontend form or display at all), ... just putting it in place for use later.